### PR TITLE
Added Getting Started and Tutorial sections under User Guides

### PR DESCRIPTION
**Pull request by thebe14**
# Summary

New sections under User Guides for the getting started guide and the tutorials. 
All tutorial pages are marked as draft.

### CODE REVIEW

File: content/en/users/getting-started/_index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Getting Started"
+linkTitle: "Getting Started"
+type: docs
+weight: 5
+draft: true
+description: >
+  Introduction to using EGI Cloud services
```
Comment (enolfc) [inline content/en/users/getting-started/_index.md:8]:
```suggestion
  Introduction to using EGI services
```

File: content/en/users/tutorials/_index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+title: "Tutorials"
+linkTitle: "Tutorials"
+type: docs
+weight: 5
+description: >
+  Turorials for common use-cases in EGI Cloud
+---
+
+The following tutorials show you how to perform common tasks in the EGI Cloud.
+These describe how to set up the EGI Cloud services needed for each use-case,
```
Comment (enolfc) [inline content/en/users/tutorials/_index.md:11]:
```suggestion
These describe how to set up the EGI services needed for each use-case,
```

File: content/en/users/tutorials/filetransfer-gridstorage/_index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: "File Transfer with grid storage"
+linkTitle: "File Transfer with grid storage"
+type: docs
+weight: 80
+description: >
+  Use EGI File Transfer to handle data in grid storage
```
Comment (enolfc) [inline content/en/users/tutorials/filetransfer-gridstorage/_index.md:7]: Are we abusing "EGI"? also "grid storage" is not something we do actually have

Comment (enolfc) [reply content/en/users/tutorials/filetransfer-gridstorage/_index.md:7]: Is `draft` missing here?

**Review by enolfc — CHANGES_REQUESTED** [review]
Mostly cosmetic issues. I think it misses several `draft: true` in various files.

File: content/en/users/tutorials/filetransfer-gridstorage/_index.md (hunk 3 above)
Comment (thebe14) [reply content/en/users/tutorials/filetransfer-gridstorage/_index.md:7]: Well, the official name of the services includes EGI, right?

Comment (thebe14) [reply content/en/users/tutorials/filetransfer-gridstorage/_index.md:7]: Added draft flag. 
As discussed in the Monday's meeting, we could make this an example on how to connect FT to S3-compatible object storage.

**Review by enolfc — APPROVED** [review]
LGTM

**Pull request merged**